classes.py

```diff
@@ -10,27 +11,76 @@ def __init__(self,
                  facebook,
```

Comment:
Im curious if we need to initialize everything or we can leave out some attributes like ruby/

Reply:
You have to indicate it as an optional argument or it will get mad.